### ISSUE TITLE
Tests fail with PICA::Data 0.36

### ISSUE THREAD

**Issue by eserte** [gbv/Catmandu-PICA#62]
The t/10-validator.t test started to fail on my smoker systems:
```
#   Failed test at t/10-validator.t line 15.
#     Structures begin differing at:
#          $got->[0]{repeated} = Does not exist
#     $expected->[0]{repeated} = '1'

#   Failed test at t/10-validator.t line 15.
#     Structures begin differing at:
#          $got->[0]{message} = 'field is not repeatable'
#     $expected->[0]{message} = 'field 021A is not repeatable'

#   Failed test at t/10-validator.t line 15.
#     Structures begin differing at:
#          $got->[0]{message} = 'field is not repeatable'
#     $expected->[0]{message} = 'field 021A is not repeatable'
# Looks like you failed 3 tests of 10.
t/10-validator.t ....... 
Dubious, test returned 3 (wstat 768, 0x300)
Failed 3/10 subtests 
```

Statistical analysis suggests that a change in PICA::Data 0.36 caused this problem (theta=1 means "bad"):
```
****************************************************************
Regression 'mod:PICA::Data'
****************************************************************
Name           	       Theta	      StdErr	 T-stat
[0='const']    	      1.0000	      0.0000	136769032566320544.00
[1='eq_0.36']  	     -1.0000	      0.0000	-45909609008771576.00

R^2= 1.000, N= 71, K= 2
****************************************************************
```

**Comment by eserte** [comment]
I still see failures with 0.26. Statistical analysis:
```
****************************************************************
Regression 'mod:PICA::Data'
****************************************************************
Name           	       Theta	      StdErr	 T-stat
[0='const']    	      0.0000	      0.0000	  27.93
[1='eq_0.37']  	      1.0000	      0.0000	222117655225212864.00

R^2= 1.000, N= 59, K= 2
****************************************************************
```

**Comment by jorol** [comment]
Fixed with v1.00
